source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProGraphicAttributes.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ArcGIS.Core.Geometry;
+using DistanceAndDirectionLibrary;
+
+namespace ProAppDistanceAndDirectionModule
+{
+    public class ProGraphicAttributes
+    {
+    }
+
+    public class LineAttributes : ProGraphicAttributes
+    {
+        Tuple<MapPoint, MapPoint, double, double> attributes;
+
+        public LineAttributes(MapPoint startPt, MapPoint endPt, double distance, double angle)
+        {
+            attributes = new Tuple<MapPoint, MapPoint, double, double>(startPt, endPt, distance, angle);
+        }
+
+        public Tuple<MapPoint,  MapPoint, double, double> GetAttributes()
```

Comment:
@rjones0 What are your thoughts of having this as a property instead of a function?